src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -200,14 +204,27 @@ class LocalBackend extends Backend with LazyLogging {
   private def stageWdlFile(wdlFile: WdlFile, hostInputsPath: Path): WdlFile = {
     val originalPath = Paths.get(wdlFile.value)
     val executionPath = hostInputsPath.resolve(originalPath.getFileName.toString)
+
+    if (tryToHardlinkInputs) {
+      try {
+        Files.createLink(executionPath, originalPath)
+      } catch {
+        case ioe: IOException =>        // recover by trying to copy rather than hardlink
+          localizePathViaCopy(originalPath, executionPath)
```

Comment:
I'd put this up above instead of the comment, the name says pretty much what the comment does